src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTarget.cs

```diff
@@ -15,7 +15,7 @@ public class LockFileTarget : IEquatable<LockFileTarget>
 
         public string RuntimeIdentifier { get; set; }
 
-        public string Name => TargetFramework + (string.IsNullOrEmpty(RuntimeIdentifier) ? "" : "/" + RuntimeIdentifier);
+        public string Name => TargetFramework.GetShortFolderName() + (string.IsNullOrEmpty(RuntimeIdentifier) ? "" : "/" + RuntimeIdentifier);
```

Comment:
Do we have roundtrip tests for this?

Reply:
Note: The spec was updated, so we no longer have assets file changes.